### PR TITLE
Escape CSV Options

### PR DESCRIPTION
**Pull request by jny986**
Added Prop to allow for the optional Escaping of CSV values

Fix for issue #127

### CODE REVIEW

**Comment by passion4IT** [comment]
@jecovier we should think about merging this pull request. This will make the life of csv export easier.

File: JsonExcel.vue [hunk 1]
```diff
@@ -205,9 +211,12 @@ export default {
...
         for (let key in item) {
-          let escapedCSV = '=\"' + item[key] + '\"'; // cast Numbers to string
-          if (escapedCSV.match(/[,"\n]/)) {
-            escapedCSV = '"' + escapedCSV.replace(/\"/g, '""') + '"';
+          let escapedCSV = item[key];
+          if (_self.escapeCsv) {
+            escapedCSV = '=\"' + escapedCSV + '\"'; // cast Numbers to string
+            if (escapedCSV.match(/[,"\n]/)) {
+              escapedCSV = '"' + escapedCSV.replace(/\"/g, '""') + '"';
+            }
```
Comment (jecovier) [inline JsonExcel.vue:219]: accept changes from @jny986 adding a prop to validate if it's necessary to escaped csv values

**Comment by jecovier** [comment]
Hey @jny986  thanks for the PR. I'm sorry to accept it so late. I abandoned this repository for a while, but now I'm catching up with the changes.

Hope you're doing well! thanks again!

**Comment by chenlevy** [comment]
Hi. I want to avoid the addition of ="" to every cell but still escape the cell in case there is comma in the value.
I can't achieve it by using escapeCSV = false because it doesn't escape the strings at all.

Maybe we should add separate property for removing ="" but still escape the values, WDYT?

**Pull request merged**